spec/services/solidus_square/webhooks/handlers/payment_created_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+RSpec.describe SolidusSquare::Webhooks::Handlers::PaymentCreated do
+  subject(:handler) { described_class.new("test") }
+
+  before do
+    allow(SolidusSquare::PaymentSyncService).to receive(:call).and_return(nil)
+    handler.call
```

Comment:
`handler.call` is the subject of the test, it should be into the `it` block

Reply:
https://github.com/nebulab/solidus_square/pull/43/commits/836e7b43554e435efd24208dab0e9526896a2d69